package.json

```
@@ -257,6 +257,15 @@
       {
         "fileMatch": "apollo.config.yaml",
         "url": "./schemas/apollo.config.schema.json"
+      },
+      {
+        "fileMatch": "apollo-router.yaml",
+        "_url": "https://github.com/apollographql/router-template/raw/main/.apollo/router_config_schema.json",
+        "url": "./schemas/router_config_schema.json"
```

I'm not sure if I should link to the schema on GitHub or a local copy here. Any thoughts @dylan-apollo?
```suggestion
        "url": "https://github.com/apollographql/router-template/raw/main/.apollo/router_config_schema.json"
```

---

If you do a local schema, you should also set up Renovate to regenerate it when new router releases come out—which would mean releasing a new version of the extension every week or two with the latest.

Or, I suppose, if we're keeping it up to date, we could also point the URL to _this_ repo instead of `router-template`?

---

We'll already have releases trickling in other places - Apollo Client release will trigger Apollo Devtools release, which will long-term trigger a release of the VSCode Devtools (or at least a PR).

I'm not too keep on doing that repeatedly with even more upstream sources, so I'd honestly rather point at another repo that is kept kinda up-to-date.

---

Oh, we could also consider publishing to https://www.schemastore.org/json/ and linking to that?

See https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md